config/default/manager_default_images.yaml

```diff
@@ -13,3 +13,5 @@ spec:
         env:
         - name: OPENSTACKCLIENT_IMAGE_URL_DEFAULT
           value: quay.io/podified-antelope-centos9/openstack-openstackclient:current-podified
+        - name: RABBITMQ_IMAGE_URL_DEFAULT
+          value: registry.redhat.io/rhosp-rhel9/openstack-rabbitmq:17.0
```

Comment:
I think we should change this to `quay.io/podified-antelope-centos9/openstack-rabbitmq:current-podified`